client/blocks/eligibility-warnings/index.jsx

```diff
@@ -53,34 +53,40 @@ export const EligibilityWarnings = ( {
 		<div className={ classes }>
 			<QueryEligibility siteId={ siteId } />
 
-			{ 'plugins' === context && ! hasBusinessPlan && ! isJetpack &&
+			{ ! hasBusinessPlan && ! isJetpack &&
 				<Banner
-					description={ translate( 'Please upgrade to install this plugin.' ) }
-					feature={ FEATURE_UPLOAD_PLUGINS }
+					description={ translate( 'Also get unlimited themes, advanced customization, no ads, live chat support, and more.' ) }
+					feature={ 'plugins' === context
+						? FEATURE_UPLOAD_PLUGINS
```

Comment:
You could get there via direct link though, e.g. `plugins/bbpress/eligibility/{ siteSlug }`.